src/gtk/window.cpp

```diff
@@ -1512,6 +1520,16 @@ bool wxWindowGTK::GTKDoInsertTextFromIM(const char* str)
 
         AdjustCharEventKeyCodes(event);
 
+        if (data.Len() == 1)
```

Comment:
> We could still synthesize the event for the last one

done

> or perhaps even for all of them

Not sure how to match all chars from input method with corresponding KeyUps. Sequential receiving is not guaranteed. Also it would need some kind of stack to store char events, not sure I am ready to do it right now. Also I even can not reproduce the case then more then 1 char received from IM.